pkg/node/kubelet.go

```diff
@@ -16,19 +16,59 @@ limitations under the License.
 package node
 
 import (
+	"context"
 	"path/filepath"
 	"strconv"
 
 	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 
 	"github.com/openshift/microshift/pkg/config"
 
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	cliflag "k8s.io/component-base/cli/flag"
 	kubeproxy "k8s.io/kubernetes/cmd/kube-proxy/app"
 	kubelet "k8s.io/kubernetes/cmd/kubelet/app"
+	kubeletoptions "k8s.io/kubernetes/cmd/kubelet/app/options"
+	kubeletconfig "k8s.io/kubernetes/pkg/kubelet/apis/config"
 )
 
-func StartKubelet(cfg *config.MicroshiftConfig) error {
-	command := kubelet.NewKubeletCommand()
+const (
+	// Kubelet component name
+	componentKubelet = "kubelet"
```

Comment:
For consistency, either we use the const in all places (like L57), or we just remove the const and use the string "kubelet".

Reply:
Another option is to use constants once you have more than one reference to the same string, that's what we did @ submariner-